internal/pkg/featureflags/feature_flags.go

```diff
@@ -40,20 +42,15 @@ const (
 )
 
 // Manager is a global feature flag manager
-var Manager featureFlagManager
+var Manager launchDarklyManager
 
-var attributes = []string{"user.resource_id", "org.resource_id", "environment.id", "cli.version", "cluster.id", "cluster.physicalClusterId"}
-
-type featureFlagManager interface {
-	BoolVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal bool) bool
-	StringVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal string) string
-	IntVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal int) int
-	JsonVariation(key string, ctx *dynamicconfig.DynamicContext, client v1.LaunchDarklyClient, shouldCache bool, defaultVal any) any
-}
+var attributes = []string{"user.resource_id", "org.resource_id", "environment.id", "cli.version", "cluster.id", "cluster.physicalClusterId", "cli.command", "cli.flags"}
 
 type launchDarklyManager struct {
 	cliClient                *sling.Sling
 	ccloudClient             func(v1.LaunchDarklyClient) *sling.Sling
+	command                  *cobra.Command
+	flags                    []string
```

Comment:
Does this disable help text? If we disable a command, but run `confluent help`, this seems like it'll still list that command, since `confluent help` is a separate command.

Reply:
This does not disable help text. I can run `confluent help` and still see the command, and I can still run the command with the `-h` flag and see help text about it. Muwei said for the output of `confluent help` either we should just prevent the top level command from showing up if it's disabled or append a `[DISABLED]` to its description, and for `-h` it doesn't matter if the help text is shown.

Reply:
> Muwei said for the output of confluent help either we should just prevent the top level command from showing up if it's disabled

The goal is to prevent any arbitrary command from showing up, not just the top-level command, right?

Reply:
Yeah, so if a child command is disabled, running its parent would result in the child now showing up or being marked as disabled.